From c96eba6692bfaf0644f92c89bbda3167de27d3b3 Mon Sep 17 00:00:00 2001
From: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Date: Thu, 3 Nov 2022 17:39:50 +0200
Subject: [PATCH 22/49] s32g3: ddr: Update timing registers

Updated the timing registers for S32G3 as part of the DDR
Tool 1.6 Update 3 release code.

Issue: ALB-9492
Upstream-Status: Pending 

Signed-off-by: Andrei Cherechesu <andrei.cherechesu@nxp.com>
Signed-off-by: Zhantao Tang <zhantao.tang@windriver.com>
---
 drivers/nxp/s32/ddr/s32g3/ddrc_cfg.c | 7 ++++---
 1 file changed, 4 insertions(+), 3 deletions(-)

diff --git a/drivers/nxp/s32/ddr/s32g3/ddrc_cfg.c b/drivers/nxp/s32/ddr/s32g3/ddrc_cfg.c
index d808fdf9a..e8a511816 100644
--- a/drivers/nxp/s32/ddr/s32g3/ddrc_cfg.c
+++ b/drivers/nxp/s32/ddr/s32g3/ddrc_cfg.c
@@ -60,7 +60,7 @@ struct regconf ddrc_cfg[] = {
 	{0x403c00c4, 0x00001000U},
 	{0x403c00d0, 0xc003061cU},
 	{0x403c00d4, 0x009e0000U},
-	{0x403c00d8, 0x00000d05U},
+	{0x403c00d8, 0x00000d00U},
 	{0x403c00dc, 0x0054002dU},
 	{0x403c00e0, 0x00330008U},
 	{0x403c00e4, 0x00100004U},
@@ -68,14 +68,14 @@ struct regconf ddrc_cfg[] = {
 	{0x403c00ec, 0x0004004fU},
 	{0x403c00f0, 0x00000000U},
 	{0x403c00f4, 0x00000779U},
-	{0x403c0100, 0x1a201b22U},
+	{0x403c0100, 0x1b203622U},
 	{0x403c0104, 0x00060633U},
 	{0x403c0108, 0x070e1517U},
 	{0x403c010c, 0x00c0c000U},
 	{0x403c0110, 0x0f04080fU},
 	{0x403c0114, 0x02040c0cU},
 	{0x403c0118, 0x01010007U},
-	{0x403c011c, 0x00000401U},
+	{0x403c011c, 0x00000402U},
 	{0x403c0120, 0x03034405U},
 	{0x403c0124, 0x0004040dU},
 	{0x403c0128, 0x001c180aU},
@@ -136,6 +136,7 @@ struct regconf ddrc_cfg[] = {
 	{0x403c0304, 0x00000000U},
 	{0x403c030c, 0x00000000U},
 	{0x403c0320, 0x00000001U},
+	{0x403c0328, 0x00000000U},
 	{0x403c0330, 0x00b03033U},
 	{0x403c0334, 0x00000000U},
 	{0x403c036c, 0x00000011U},
-- 
2.25.1

